test/jdk/jdk/jfr/event/io/TestSocketAdapterEvents.java

```diff
@@ -144,6 +150,8 @@ private static void testConnectException() throws Throwable {
                 try (SocketChannel sc = SocketChannel.open()) {
                     Socket s = sc.socket();
                     s.connect(addr);
+                    // unexpected, abandon the test
+                    return false;
```

Comment:
The main issue with using the ephemeral port range is that you might manage to connect to a server opened by another test, and that might cause the other test to fail if it's not expecting connections to get closed.

If instead you use ports in the IANA reserved port range - at least you know that you won't connect to other tests running on the same machine.

Have you tried to connect to port 225 for instance, and increase the port number up to 241 in case you still manage to connect?

Ports 225-241 are reserved by IANA - so there should be nobody listening there. I had some trouble on windows 2016 with port 47, but hopefully ports 225-241 will not have the same issue.

Reply:
I've changed the exception tests to use the port range you suggested.